src/WebSdk/Web/Targets/Sdk.Server.targets

```diff
@@ -14,6 +14,8 @@ Copyright (c) .NET Foundation. All rights reserved.
   <!-- We can't add it here because the OutputPath and other msbuild properties are not evaluated.-->
   <!--<Import Sdk="Microsoft.NET.Sdk.Publish" Project="ImportPublishProfile.targets" />-->
 
+  <Import Sdk="Microsoft.NET.Sdk.Web.ProjectSystem" Project="..\targets\Microsoft.NET.Sdk.Web.BeforeCommon.targets" />
```

Comment:
Out of curiosity, why a new target? Why not do this inline like AddRazorSupportForMvc?

Reply:
It is setting up some "infrastructure" for a place where we can add more settings that need to be defined between the .csproj and Microsoft.NET.Sdk's targets. A lot of the value is defining intent from the name of the file (BeforeCommon.targets). We have the same thing in the NET.Sdk here:

https://github.com/dotnet/sdk/blob/main/src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.BeforeCommon.targets

This way everything doesn't get piled into `Sdk.targets`.